### ISSUE TITLE
Implement

### ISSUE THREAD

**Issue by kam1107** [pykaldi/pykaldi#61]
Hi, I noticed that nnet3 is implement in pykaldi. Is there any chance to also include nnet3bin (kaldi)? Furthermore, can xvector pipeline be implemented in pykaldi?  Thank you very much.

**Comment by dogancan** [comment]
We don't have any plans of implementing executables replicating the functionality of Kaldi command-line programs. Do you have a specific need that is not met by the Kaldi executables?

The xvector testing pipeline (not training) can (probably) be implemented in PyKaldi, but I don't know what it would bring to the table beyond the native implementation in Kaldi. Again, do you have a specific need that is not met by the Kaldi xvector pipeline?

**Comment by kam1107** [comment]
We plan to fine tune the network, or make some attempts on implementing a different loss function (e.g. center loss) to substitute softmax. We thought Python is more friendly for experiments like these.

**Comment by dogancan** [comment]
What you are saying makes sense but I don’t quite follow how reimplementing kaldi executables in pykaldi would help you achieve that.

**Comment by dogancan** [comment]
Hey I am closing this issue but feel free to reopen it if you think there is something we can do to help.
